src/assets/i18n/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface I18n {
```

Comment:
What about creating a string enum with the values you always want populated for each language and then making I18n a AudificationLabelEnum (or some better name) : string type? Then you could use the enum to select values in the d3 and line-chart code below.

Reply:
Great idea! I'll try that way.